From 8fcb4ed7983d747088fd188a303d9d0ceb6ab679 Mon Sep 17 00:00:00 2001
From: Stefan Dirsch <sndirsch@suse.de>
Date: Mon, 14 Apr 2014 11:25:33 +0200
Subject: [PATCH] Back to previous "ru(phonetic)" keyboard layout.

This reverts commit 09989fd4b68948be51567b2c78f3ce4e2f75c55c

  Fix Russian phonetic layout

    According to
      https://bugzilla.novell.com/show_bug.cgi?id=498304
    the mapping of the Russian phonetic layout doesn't match what
    is shown in:
      http://en.wikipedia.org/wiki/Keyboard_layout#Russian

Apparently I've implemented the most prominently shown russian
phonetic keyboard on Wikipedia, which at the same is at least being
used and also appears at most unlogic. The initial request came from
HP. Sources are no longer known. So let's revert this one.

Signed-off-by: Stefan Dirsch <sndirsch@suse.de>
(cherry picked from commit e8081e63d7218c267f767f0310e4dc7513ca8643)

Conflicts:
	symbols/ru
---
 symbols/ru | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/symbols/ru b/symbols/ru
index 043e011..7c47b96 100644
--- a/symbols/ru
+++ b/symbols/ru
@@ -179,7 +179,7 @@ xkb_symbols "phonetic" {
     key	<LatA> {	[      Cyrillic_a,	Cyrillic_A	]	};
     key	<LatW> {	[     Cyrillic_ve,     Cyrillic_VE	]	};
     key	<LatC> {	[    Cyrillic_tse,    Cyrillic_TSE	]	};
-    key	<LatX> {	[     Cyrillic_ha,     Cyrillic_HA	]	};
+    key	<LatX> {	[ Cyrillic_softsign, Cyrillic_SOFTSIGN	]	};
     key	<LatD> {	[     Cyrillic_de,     Cyrillic_DE	]	};
     key	<LatE> {	[     Cyrillic_ie,     Cyrillic_IE	]	};
     key	<LatV> {	[    Cyrillic_zhe,    Cyrillic_ZHE	]	};
@@ -188,7 +188,7 @@ xkb_symbols "phonetic" {
     key	<LatR> {	[     Cyrillic_er,     Cyrillic_ER	]	};
     key	<LatN> {	[     Cyrillic_en,     Cyrillic_EN	]	};
     key	<LatB> {	[     Cyrillic_be,     Cyrillic_BE	]	};
-    key	<LatH> {	[     Cyrillic_che,     Cyrillic_CHE	]	};
+    key	<LatH> {	[     Cyrillic_ha,     Cyrillic_HA	]	};
     key	<LatG> {	[    Cyrillic_ghe,    Cyrillic_GHE	]	};
     key	<LatY> {	[   Cyrillic_yeru,   Cyrillic_YERU	]	};
     key	<LatM> {	[     Cyrillic_em,     Cyrillic_EM	]	};
@@ -201,7 +201,7 @@ xkb_symbols "phonetic" {
     key	<LatP> {	[     Cyrillic_pe,     Cyrillic_PE	]	};
 
     key	<AD11> {	[    Cyrillic_sha,    Cyrillic_SHA	]	};
-    key	<AE12> {	[ Cyrillic_softsign, Cyrillic_softsign  ]	};
+    key	<AE12> {	[    Cyrillic_che,    Cyrillic_CHE	]	};
     key	<AD12> {	[  Cyrillic_shcha,  Cyrillic_SHCHA	]	};
     key	<BKSL> {	[      Cyrillic_e,	Cyrillic_E	]	};
 };
-- 
2.1.0

